src/components/tl/dpu/tl_dpu_context.c

```diff
@@ -0,0 +1,246 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2021.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#include "tl_dpu.h"
+#include "tl_dpu_coll.h"
+
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <netinet/ip.h>
```

Comment:
do we need configure detection for netinet/in.h netinet/ip.h netdb.h or are those headers always available?